### PR TITLE
fix #550 - download shiny server from new download location

### LINKED ISSUE #550: Script install_shiny_server.sh when using custom SHINY_SERVER_VERSION 1.5.19.995 refers to https://download3.rstudio.org/ubuntu-14.04/x86_64/shiny-server-1.5.19.995-amd64.deb which gives 404
### Container image name

rocker/ml-verse:4.2.1

### Container image digest

rocker/ml-verse@sha256:c793fb4b5a3da479b81e9429e565d1f3fa5d6a221786c4dfbcf0deec764d07d3

### What operating system are you seeing the problem on?

Linux

### System information

Docker version 20.10.19, build d85ef84                                          
Kernel 5.4.0-128-generic #144-Ubuntu SMP Tue Sep 20 11:00:04 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### Bug description

It may be related to setting the SHINY_SERVER_VERSION to 1.5.19.995, causing https://github.com/rocker-org/rocker-versioned2/blob/master/scripts/install_shiny_server.sh#L38 to attempt to download from `ubuntu-14.04` directory which does not have this file:

#0 28.17 https://download3.rstudio.org/ubuntu-14.04/x86_64/shiny-server-1.5.19.995-amd64.deb:
#0 28.17 2022-10-18 09:35:51 ERROR 404: Not Found.

### How to reproduce this bug?

```sh
Use a Dockerfile with


FROM rocker/ml-verse:4.2.1
ENV SHINY_SERVER_VERSION 1.5.19.995
RUN /rocker_scripts/install_shiny_server.sh
```

### PR DESCRIPTION
**Pull request by mskyttner**

### CODE REVIEW

**Comment by mskyttner** [comment]
I think the [check that failed](https://github.com/rocker-org/rocker-versioned2/actions/runs/3276109308/jobs/5393521894) is related to something else, maybe a temporary 404 download issue when running the script to install rstudio, since this file is now available:

```
RUN /test.sh install_rstudio.sh daily

 #9 11.08 --2022-10-18 21:04:14--  https://rstudio.org/download/latest/daily/server/jammy/rstudio-server-latest-amd64.deb
404 Not Found
#9 11.28 2022-10-18 21:04:15 ERROR 404: Not Found.
```

**Comment by eitsupi** [comment]
> I think the [check that failed](https://github.com/rocker-org/rocker-versioned2/actions/runs/3276109308/jobs/5393521894) is related to something else, maybe a temporary 404 download issue when running the script to install rstudio, since this file is now available:

Yes, this issue is not related the script. rstudio/rstudio#12173

File: scripts/install_shiny_server.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 set -e
 
 SHINY_SERVER_VERSION=${1:-${SHINY_SERVER_VERSION:-latest}}
+UBUNTU_VERSION=${1:-${UBUNTU_VERSION:-18.04}}
```
Comment (eitsupi) [inline scripts/install_shiny_server.sh:5]: Could you remove this line like I commented?
https://github.com/rocker-org/rocker-versioned2/issues/550#issuecomment-1282621585

**Review by eitsupi — CHANGES_REQUESTED** [review]

File: scripts/install_shiny_server.sh (hunk 1 above)
Comment (mskyttner) [reply scripts/install_shiny_server.sh]: Yes

**Review by eitsupi — APPROVED** [review]
Thanks for working on this!

**Review by noamross — APPROVED** [review]

**Pull request merged**